### PR TITLE
[Port] Add sac2016 suffix to Nano Server Dockerfile folder name.

### PR DESCRIPTION
**Pull request by ravimeda**
Port changes from https://github.com/dotnet/dotnet-docker-nightly/pull/486

Add `-sac2016` suffix to the Nano Server folder name with the corresponding Dockerfile. Updated readme.

### CODE REVIEW

File: 1.1/sdk/nanoserver/Dockerfile [hunk 1]
```diff
@@ -1,23 +0,0 @@
-# escape=`
```
Comment (MichaelSimons) [inline 1.1/sdk/nanoserver/Dockerfile]: Why was this file removed?  In #334 it was mentioned the 1.0/sdk file should be removed not the 1.1/sdk!

File: build-and-test.ps1 [hunk 2]
```diff
@@ -48,7 +48,7 @@ $manifestRepo.Images |
                 }
                 $formattedTags = $qualifiedTags -join ', '
                 Write-Host "--- Building $formattedTags from $dockerfilePath ---"
-                Invoke-Expression "docker build $optionalDockerBuildArgs -t $($qualifiedTags -join ' -t ') $dockerfilePath"
+                #Invoke-Expression "docker build $optionalDockerBuildArgs -t $($qualifiedTags -join ' -t ') $dockerfilePath"
```
Comment (MichaelSimons) [inline build-and-test.ps1]: Looks like your changes to do some testing were inadvertently checked in.  (More below)

File: 1.1/sdk/nanoserver/Dockerfile (hunk 1 above)
Comment (ravimeda) [reply 1.1/sdk/nanoserver/Dockerfile]: Fixed.

File: build-and-test.ps1 (hunk 2 above)
Comment (ravimeda) [reply build-and-test.ps1]: Fixed.

**Comment by ravimeda** [comment]
Looks like merge did not happen correctly. I'll push a new commit.

**Comment by ravimeda** [comment]
@dotnet-bot test Debian - 1. Dockerfiles please
